Makefile.PL

```diff
@@ -0,0 +1,40 @@
+use 5.010;
+use strict;
+use warnings;
+use ExtUtils::MakeMaker;
+
+WriteMakefile(
+    NAME             => 'Mojo::WebSocketProxy',
+    VERSION_FROM     => 'lib/Mojo/WebSocketProxy.pm',
+    AUTHOR           => 'Binary.com (binary@cpan.org)',
+    ABSTRACT         => 'Perl implementation of websocket-JSON-RPC 2.0 proxy',
+    PL_FILES         => {},
+    MIN_PERL_VERSION => 5.010,
+    BUILD_REQUIRES   => {
+        'Test::More'       => 0,
+        'Test::MockModule' => 0,
+        'Test::MockObject' => 0,
+        'JSON'             => 0,
+    },
+    PREREQ_PM => {
+        'Test::Simple' => 0.44,
+        'Mojolicious'  => 0,
```

Comment:
If I compare this list with the `before_install` section in `.travis.yml`, I get the impression that this here is not complete.